Resources/Prototypes/Datasets/tips.yml

```diff
@@ -83,27 +83,27 @@
   - "Spears can be tipped with chemicals, and will inject a few units every time you hit someone with them directly."
   - "You can make spears with reinforced glass, plasma glass, or uranium glass shards to enhance their damage."
   - "Thrown spears deal extra damage! Beware, however, as throwing them too much will end up breaking them."
-  - "Shivs and spears that are made with uranium glass can deal radiation damage, which requires medicine to properly treat."
-  - "You can throw crafted bolas at people to slow them down, letting you follow up on them for an easier kill or get away."
+  - "All forms of toxin damage are fairly difficult to treat, and usually involve the use of chemicals or other inconvienent methods. You can use this to your advantage in combat."
+  - "You can throw crafted bolas at people to slow them down, letting you follow up on them for an easier kill or getaway."
   - "You can put napalm in a backpack water tank to make a flamethrower."
-  - "Most jobs have alternate uinforms in their drobe machine. Don't be afraid to try out a new look!"
-  - "As any other job other than salvage, it is usually wise to avoid using hardsuits unless you need them, since the slowdown they produce is often much more detrimental than any resistances or spaceproofing you may get."
+  - "Some jobs have alternate uinforms in their respective drobe vendors. Don't be afraid to try out a new look!"
+  - "Speed is almost everything in combat. Using hardsuits just for their armor is usually a terrible idea unless the resistances it provides are geared towards combat, or you're not planning to go head-first into the fray."
   - "Just because a job can't be a traitor at the beginning of a round doesn't mean that they'll never be a traitor."
-  - "Syndicate gas masks will block flashes, meaning that Nuclear Operatives and other dubious individuals wearing them can't be flashed."
+  - "Syndicate gas masks will both provide welding protection and block flashes. Think twice before trying to flash a Nuclear Operative!"
   - "Demoman takes skill."
   - "You can spray a fire extinguisher, throw items or fire a gun while floating through space to give yourself a minor boost. Simply fire opposite to where you want to go."
-  - "You can drag other players onto yourself to open the strip menu, letting you remove their equipment or force them to wear something. Note that exosuits or helmets will block your access to the clothing beneath them, and that certain items take longer to strip or put on than others."
+  - "You can drag other players onto yourself to open the strip menu, allowing you to remove their equipment or force them to wear something. Note that exosuits or helmets will block your access to the clothing beneath them, and that certain items take longer to strip or put on than others."
```

Comment:
More words is not always better

Reply:
I changed this so the the scentence flowed better. "Letting you remove their equipment or force them to wear something" doesn't feel quite gramatically correct, but I may be wrong.